_includes/link-list.html

```diff
@@ -1,14 +1,52 @@
 <dl class="link-list">
-{% for link in include.list %}
+{% for link in include.links %}
+
+{% if link.href_snapshot %}
+	{% assign linkURL = link.href_snapshot %}
+{% elsif link.snapshot %}
+	{% assign linkURL = 'http://archive.is/' | append: link.snapshot %}
+{% else %}
+	{% assign linkURL = link.href %}
+{% endif %}
 
   <dt class="single-line">
 	<i class="fa fa-fw {{ link.icon }}"></i>
-	{% if link.href_snapshot %}
-	  <a href="{{ link.href_snapshot }}" {{ link.data }}>{{ link.title }}</a>
+	{% if link.type == "Review" %}
+		{% if link.rating %} 
+			{% assign score = link.rating | split: "/" %} 
+			{% assign score_top = score[0] | times:1.0 %}
+			{% assign score_bottom = score[1] | times: 1.0 %} 
+			{% assign score_rating = score_top | divided_by:score_bottom %} 
+			{% assign score_scaled = score_rating | times:5 %} 
+		{% endif %}
+
+		{% if link.stars %} 
+			{% assign score_scaled = link.stars %}
+		{% endif %}
+
+		<!-- Produce stars -->
+		{% for star in (1..score_scaled) %} 
+			<i class="fa fa-star {% increment star %}"></i> 
+		{% endfor %}
+		{% assign score_rounded = score_scaled | round %}
+		{% if score_rounded != score_scaled %}
+			<i class="fa fa-star-half-o"></i> 
+		{% endif %}
+		{% for nostar in (score_rounded..4.9) %}
+			<i class="fa fa-star-o {% increment nostar %}"></i> 
+		{% endfor %}
+
+		<strong>{{ link.publisher }}</strong> - <em>"{{ link.quote }}" - <a href="{{ linkURL }}">Full Review</a></em> 
+		<a href="{{ linkURL }}" target="_blank"><small><i class="fa fa-external-link"></i></small></a>
+
 	{% else %}
-	  <a href="{{ link.href }}" {{ link.data }}>{{ link.title }}</a>
-	{% endif %}
+
+  <a href="{{ linkURL }}" {{ link.data }}>{{ link.title }}</a>
```

Comment:
Can we remove changes to these lines, looks like extra whitespace and removal of indents.